dashboard/tests/test_dashboard.py

```diff
@@ -705,5 +705,18 @@ def test_gcs_check_alive(fast_gcs_failure_detection, ray_start_with_dashboard):
     assert dashboard_proc.wait(10) == 255
 
 
+@pytest.mark.skipif(
+    os.environ.get("RAY_MINIMAL") != "1",
+    reason="This test only works for minimal installation.",
+)
+def test_dashboard_does_not_depend_on_serve():
+    """Check that the dashboard can start without Serve."""
+
+    with pytest.raises(ImportError):
+        from ray import serve  # noqa F401
+
+    ray.init(include_dashboard=True)
+
```

Comment:
ray dashboard doesn't work with minimal install. ray dashboard only works when ray[default] is installed

Reply:
Got it, I added a script to perform the test with a default installation.

Reply:
can you check regular dashboard process works (e.g. /api/snapshot) and serve doesn't?